src/Infrastructure/LeanCode.ViewRenderer.Razor/RazorViewRendererServiceCollectionExtensions.cs

```diff
@@ -10,7 +10,9 @@ public static IServiceCollection AddRazorViewRenderer(
         RazorViewRendererOptions config
     )
     {
-        services.TryAddSingleton<IViewRenderer>(_ => new RazorViewRenderer(config));
-        return services;
+        return services.AddSingleton<IViewRenderer>(sp => new RazorViewRenderer(
+            config,
+            sp.GetRequiredService<ILogger<RazorViewRenderer>>()
+        ));
```

Comment:
seems out of scope

Reply:
Indeed, but it was a small inconsistency I wanted to fix along the way.